packages/tempstore/tempStore.js

```diff
@@ -293,8 +293,16 @@ FS.TempStore.createWriteStream = function(fileObj, options) {
     setObj['keys.' + chunkNum] = result.fileKey;
     tracker.update(selector, {$set: setObj});
 
+
+    var temp = tracker.findOne(selector);
+
+    if(!temp){
+      FS.debug && console.log('NOT FOUND FROM TEMPSTORE => EXIT (REMOVED)');
+      return;
+    }
+
     // Get updated chunkCount
-    var chunkCount = FS.Utility.size(tracker.findOne(selector).keys);
+    var chunkCount = FS.Utility.size(temp.keys);
```

Comment:
This part is pretty straight forward. What is unclear is when we get in the condition where `temp` is empty. @louis49 do you have something to add here?